### ISSUE TITLE
Twitter profile image doesn't show

### ISSUE THREAD

**Issue by shibayu36** [metacpan/prepan#42]
Twitter profile image path was changed in Twitter API v1.1

**Comment by oalders** [comment]
Yep "The Twitter REST API v1 is no longer active. Please migrate to API v1.1. https://dev.twitter.com/docs/api/1.1/overview."
